admin_guide/image_signatures.adoc

```diff
@@ -152,6 +152,24 @@ $ oc adm verify-image-signature \
     --save
 ----
 
+[NOTE]
+====
+If the `oc adm verify-image-signature` command returns an `x509: certificate signed by unknown authority` error, you might need to add the registry's certificate authority (CA) to the list of CAs trusted on the system. You can do this by performing the following steps:
+
+. Copy the CA certificate to the *_/etc/pki/ca-trust/source/anchors/_* directory. For example:
++
+----
+# cp /etc/docker/certs.d/docker-registry.default.svc\:5000/node-client-ca.crt \
+    /etc/pki/ca-trust/source/anchors/
+----
```

Comment:
This node-client-ca.crt is from cluster, user need to copy the crt file to the system where oc is running on, maybe need scp.